src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -36,6 +36,7 @@ Microsoft.CodeAnalysis.Diagnostics.Telemetry.AnalyzerTelemetryInfo.OperationBloc
 Microsoft.CodeAnalysis.Diagnostics.Telemetry.AnalyzerTelemetryInfo.OperationBlockEndActionsCount.set -> void
 Microsoft.CodeAnalysis.Diagnostics.Telemetry.AnalyzerTelemetryInfo.OperationBlockStartActionsCount.get -> int
 Microsoft.CodeAnalysis.Diagnostics.Telemetry.AnalyzerTelemetryInfo.OperationBlockStartActionsCount.set -> void
+Microsoft.CodeAnalysis.IMethodSymbol.RefKind.get -> Microsoft.CodeAnalysis.RefKind
 Microsoft.CodeAnalysis.IMethodSymbol.ReturnsByRefReadonly.get -> bool
```

Comment:
>ReturnsByRefReadonly [](start = 37, length = 20)

If RefKind stays, we may want to remove "ReturnsByRefReadonly". And if we do, perhaps before 15.5 - should we add a tracking issue?

Reply:
I do thin RefKind is a good idea. Just makes ReturnsByRefReadonly redundant and since that one is also new, we can remove it.

---
In reply to: [141178011](https://github.com/dotnet/roslyn/pull/22285#discussion_r141178011) [](ancestors = 141178011)

Reply:
I do think that the redundency of `ReturnsByRefReadOnly` is the same as `ReturnsByRef`. Why should one exist without the other?

---
In reply to: [141178222](https://github.com/dotnet/roslyn/pull/22285#discussion_r141178222) [](ancestors = 141178222,141178011)